MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -29,7 +29,7 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
 
     private let attributes = ["age": 40]
```

Comment:
do we still need this? If not, can you remove it?

Reply:
With the latest changes, it is not needed anymore. Removed.